src/Microsoft.Language.Xml.Tests/TestApi.cs

```diff
@@ -13,9 +13,24 @@ public void TestAttributeValue()
             Assert.Equal("", attributeValue);
         }
 
+        /// <summary>
+        /// 2.2.12 [XML] Section 3.3.3, Attribute-Value Normalization
+        /// <seealso href="https://learn.microsoft.com/en-us/openspecs/ie_standards/ms-xml/389b8ef1-e19e-40ac-80de-eec2cd0c58ae" />
+        /// </summary>
+        [Fact]
+        public void TestAttributeValueNormalization()
+        {
+            const string AllWhitespace = " \n\t";
+            var xml = $"<ContentControl Content=\"{AllWhitespace}X{AllWhitespace}\" />";

```

Comment:
Can you please minimize the repro and use A, B or X instead of longer words? ContentControl has semantics that is not necessary here.